lnwallet/channel.go

```diff
@@ -4011,6 +4036,9 @@ func (lc *LightningChannel) ProcessChanSyncMsg(
 					),
 					CommitSig: commitSig,
 					HtlcSigs:  htlcSigs,
+					RemoteNonce: &lnwire.RemoteMusig2Nonce{
```

Comment:
I think this will send a remotenonce (all 0s) even if this is not a taproot channel